geomstats/_backend/autograd/__init__.py

```diff
@@ -136,6 +135,15 @@
 power = _box_binary_scalar(target=_np.power)
 
 
+def squeeze(x, axis=None):
```

Comment:
We should probably do the same in `numpy.squeeze`.

Reply:
True, done!